bin/md2short.sh

```diff
@@ -69,16 +75,22 @@ if [[ -f "$OUTFILE" && -z "$OVERWRITE" ]]; then
   done
 fi
 
+if [[ ! -z "$MODERN" ]]; then
```

Comment:
Please use -n instead of ! -z.

See: https://www.shellcheck.net/wiki/SC2236

Reply:
Done.